tasks/set_parameter_contexts.yml

```diff
@@ -0,0 +1,19 @@
+---
+- name: Install jq
+  include: install_jq.yml
+- name: Get existing Parameter Contexts
+  script:
```

Comment:
I can't figure out this command, https://docs.ansible.com/ansible/latest/modules/script_module.html

`yaml-lint` complains when using `shell` here, it wants a condition to be added, so if someone knows the condition I'm fine moving back to that/

the `chdir` doesn't seem to be taking effect, since this is failing w/ a can't find `bin/cli.sh`. Having `chdir` at the same level as `script` also didn't seem to work.

I'm probably missing something silly, please help me out.

Reply:
I think this is because `script` attempts to find the script locally and not on the remote instance.  I changed this to `command` instead - the linter seems okay with that.